### PR TITLE
fix(argo-rollouts): Add traefikservices resource in role

### PR DESCRIPTION
**Pull request by JuanRamino**
Signed-off-by: juan ramino <juan.ramino@gmail.com>

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [ ] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [ ] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [ ] Any new values are backwards compatible and/or have sensible default.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

Changes are automatically published when merged to `main`. They are not published on branches.

### CODE REVIEW

**Comment by jmeridth** [comment]
Whoops.  Gonna let tests run first.

**Comment by jmeridth** [comment]
@JuanRamino chart needs a version bump.

File: charts/argo-rollouts/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: v1.3.1
 description: A Helm chart for Argo Rollouts
 name: argo-rollouts
-version: 2.21.1
+version: 2.21.2
```
Comment (mkilchhofer) [inline charts/argo-rollouts/Chart.yaml:5]: See some lines below, please update the line for the changelog generation

File: charts/argo-rollouts/templates/controller/role.yaml [hunk 2]
```diff
@@ -150,4 +150,15 @@ rules:
...
+- apiGroups:
+  - traefik.containo.us
+  resources:
+  - traefikservices
+  verbs:
+  - list
+  - watch
+  - get
+  - update
+  - patch
```
Comment (mkilchhofer) [inline charts/argo-rollouts/templates/controller/role.yaml:163]: Can you test with the upstream verbs only (watch, get, update)?

https://github.com/argoproj/argo-rollouts/blob/v1.3.2/manifests/role/argo-rollouts-clusterrole.yaml#L224-L231

**Review by mkilchhofer — CHANGES_REQUESTED** [review]

**Comment by mkilchhofer** [comment]
This was fixed/implemented in #1774 already and thus closing this PR here. Feel free to reopen if needed.
Thanks for your contribution anyways. 🙏

**Comment by JuanRamino** [comment]
Seems fine! Glad to contribute anyway. Thanks

**Pull request closed without merging**